toolchains/docker/toolchain.bzl

```diff
@@ -68,8 +68,8 @@ docker_toolchain = rule(
         ),
         "client_config": attr.string(
             default = "",
-            doc = "A custom directory or a Bazel label for the docker client config.json. " +
-                  "If DOCKER_CONFIG is not specified, the value of the " +
+            doc = "A custom directory for the docker client config.json. If " +
```

Comment:
Would you agree the documentation for this attribute reads strangely? I don't really understand what it's trying to say. Might be good to update this?

Reply:
How about now?

Reply:
Much better, though the second to last sentence is missing an "If" at the beginning I believe.